perllib/FixMyStreet/Cobrand/Smidsy.pm

```diff
@@ -292,7 +292,13 @@ sub prettify_incident_dt {
     };
 }
 
-sub send_questionnaires {
+sub send_questionnaire {
+    my ($self, $report) = @_;
+
+    my $severity = $report->get_extra_metadata('severity');
+    my $participant = $report->get_extra_metadata('participants');
+
+    return 1 if $severity > 10 && $participant =~ /(car|motorcycle|hgv|other)/;
```

Comment:
Given we're using this in multiple places it might be nice to stick it in a cobrand method rather than duplicating this logic.